client/securedrop_client/gui/main.py

```diff
@@ -75,6 +80,13 @@ def __init__(
         self.main_pane.setLayout(layout)
         self.left_pane = LeftPane()
         self.main_view = MainView(self.main_pane, app_state)
```

Comment:
(I think I've introduced a bug here that I need to fix - there's a logline about attempting to apply a layout to MainView when a layout has already been applied. It doesn't have any ill effects, but it's on my radar to find + fix before this PR is ready for review.)

Reply:
fixed in [e4bb9d9](https://github.com/freedomofpress/securedrop-client/pull/2252/commits/e4bb9d9e32ba1a72e241aa05989e4fd3cb86b8ed)